js/ui/action_button/action_button_base.js

```diff
@@ -0,0 +1,223 @@
+import $ from "../../core/renderer";
+import config from "../../core/config";
+import { extend } from "../../core/utils/extend";
+import eventsEngine from "../../events/core/events_engine";
+import eventUtils from "../../events/utils";
+import clickEvent from "../../events/click";
+import iconUtils from "../../core/utils/icon";
+import swatch from "../widget/swatch_container";
+import ActionButtonItem from "./action_button_item";
+import themes from "../themes";
+
+const FAB_MAIN_CLASS = "dx-fa-button-main";
+const FAB_ICON_CLASS = "dx-fa-button-icon";
+const FAB_CLOSE_ICON_CLASS = "dx-fa-button-icon-close";
+
+let actionButtonBase = null;
+
+const ActionButtonBase = ActionButtonItem.inherit({
+    _getDefaultOptions() {
+        return extend(this.callBase(), {
+            icon: config().actionButtonConfig.icon,
+            closeIcon: config().actionButtonConfig.closeIcon,
+            position: config().actionButtonConfig.position,
+            maxActionButtonCount: config().actionButtonConfig.maxActionButtonCount,
+            onClick: null,
+            actions: [],
+            visible: true,
+            activeStateEnabled: true,
+            hoverStateEnabled: true,
+            indent: 56,
+            childIndent: 36
+        });
+    },
+
+    _defaultOptionsRules() {
+        return this.callBase().concat([
+            {
+                device: function() {
+                    return themes.isMaterial();
+                },
+                options: {
+                    indent: 80,
+                    childIndent: 56
+                }
+            }
+        ]);
+    },
+
+    _render() {
+        this.element().addClass(FAB_MAIN_CLASS);
+        this.callBase();
+        this._renderIcon();
+        this._renderCloseIcon();
+        this._renderInkRipple();
+        this._renderClick();
+    },
+
+    _renderContent() {
```

Comment:
This method just call `callBase`
Why not just remove this overload if you do not override or extend the existed method implementation?